packages/protocol-kit/src/utils/transactions/utils.ts

```diff
@@ -52,8 +53,8 @@ export async function standardizeSafeTransactionData({
   }
 
   let safeVersion: SafeVersion
-  if (predictedSafe?.safeDeploymentConfig?.safeVersion) {
-    safeVersion = predictedSafe?.safeDeploymentConfig.safeVersion
+  if (!!predictedSafe) {
```

Comment:
Why not `if (predictedSafe)`?
I understand that `!!` converts into a boolean with the same value, but I also find it a little bit more difficult to read than just having `if (predictedSafe)`

Reply:
done!